src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs

```diff
@@ -367,7 +367,7 @@ public void should_parse_language_slovak(string postTitle)
             var result = Parser.Parser.ParseMovieTitle(postTitle);
             result.Languages.Should().BeEquivalentTo(Language.Slovak);
         }
-
+        
```

Comment:
Get rid of this whitespace

Reply:
I've removed all whitespaces and blank lines, yet other checks still fail, thanks @Qstick

Reply:
Single blank lines are fine, and needed in some cases. Whitespace is not